samples/features/sql-big-data-cluster/connectivity/webhdfs-java-client/login.conf

```
@@ -0,0 +1,11 @@
+com.sun.security.jgss.login {
+    com.sun.security.auth.module.Krb5LoginModule required client=TRUE useTicketCache=true debug=true principal="admin";
+ };
+ 
+ com.sun.security.jgss.initiate {
+    com.sun.security.auth.module.Krb5LoginModule required client=TRUE useTicketCache=true debug=true principal="admin";
```

```suggestion
    com.sun.security.auth.module.Krb5LoginModule required client=TRUE useTicketCache=true debug=false principal="admin";
```

Debug can be set to true only when needed.